api/python/gdh.py

```diff
@@ -0,0 +1,60 @@
+# Depends: requests pandas
+# You may already have these installed, otherwise install using pip:
+#     pip install requests pandas
+
+# Example invocations:
+# >>> from gdh import cases
+# >>> c1 = cases("country:'New Zealand'", "API KEY HERE")
+# >>> c2 = cases("country:'New Zealand' admin1:Auckland", "API KEY HERE")
+#
+# You can use the cases_cached() function to cache the cases locally.
+# This is useful for rerunning the script which will then use the
+# cached version.
+
+import os
+import io
+import json
+import hashlib
+
+import requests
+import pandas as pd
+
+GH = "https://data.covid-19.global.health"
+downloadAsync = "/api/cases/downloadAsync"
+
+def cases(query, apikey, server=GH):
+    res = requests.post(
+        f"{server}{downloadAsync}",
+        data=json.dumps({"format": "csv", "query": query}),
```

Comment:
"csv" might be better as a default `kwarg` value to extend this function to other formats

Reply:
`cases()` is about providing the case data as a dataframe without regard to the underlying format, I'll add a function to download in the specific format requested separately.